bedrock/mozorg/templates/mozorg/moss/base.html

```diff
@@ -49,8 +49,6 @@ <h1 class="masthead-logo">
     </nav>
   </div>
 {% endif %}
-  <div id="header-image">{% block header_image %}{% endblock %}</div>
```

Comment:
We don't want to remove this from the base template, as it's used to display header images on other `/moss` pages, e.g. https://www.mozilla.org/en-US/moss/secure-open-source/